tests/settings/ConfigurationGatheringTest.py

```diff
@@ -63,13 +63,13 @@ def test_gather_configuration(self):
 
         self.assertEqual(len(local_bears['default']), 0)
 
-    def test_default_coafile_parsing(self):
+    def test_base_coafile_parsing(self):
```

Comment:
The test `test_default_coafile_presence` will be similar to this one just replace `base_coafile` with `default_coafile`